src/org/thoughtcrime/securesms/ConversationFragment.java

```diff
@@ -393,7 +393,8 @@ public void scrollToBottom() {
     }
 
     void setLastSeen(long lastSeen) {
-        getListAdapter().setLastSeen(lastSeen);
+        ConversationAdapter listAdapter = getListAdapter();
```

Comment:
@r10s did a PR to fix this, you two should compare which solution is better and revert this or close #2512

Reply:
Both seem to be fine, but @r10s' solution looks more "careful", so let's go with this one